bundles/org.openhab.binding.homeconnect/src/main/resources/assets/js/homeconnect.js

```diff
@@ -134,6 +135,23 @@
             responseBodyElement.addClass('text-muted')
         }
 
+        titleBadgeElement.empty();
+        if (request.homeConnectResponse) {
+            var statusCode = request.homeConnectResponse.code;
+            titleBadgeElement.text(statusCode);
+            titleBadgeElement.removeClass('badge-success');
+            titleBadgeElement.removeClass('badge-danger');
+            titleBadgeElement.removeClass('badge-warning');
+
+            if (statusCode >= 300 && statusCode != 404) {
```

Comment:
You have a special case for 404 which is considered only as a warninbg ?

Reply:
Yes that is correct. It is "ok" if we receive a 404. For example if a program isn't available. 
It's the same logic as in the main request view.